packages/nodes-base/nodes/MongoDb/mongo.node.utils.ts

```diff
@@ -18,12 +18,20 @@ import { get, set } from 'lodash';
  *
  * @param {ICredentialDataDecryptedObject} credentials MongoDB credentials to use, unless conn string is overridden
  */
-function buildParameterizedConnString(credentials: IMongoParametricCredentials): string {
+function buildParameterizedConnString(
+	credentials: IMongoParametricCredentials,
+): string {
+	let connectionString = `mongodb+srv://${credentials.user}:${credentials.password}@${credentials.host}`;
```

Comment:
Hey @brianinoa I had to undo this change since it wouldn't work for me - port, although not mandatory, cannot be unset (if I left it empty it would be filled again). Also it won't accept `localhost` as host when using `mongo+srv` as protocol.

Could you please rever this part of the change? The rest is looking great.

Reply:
Great catch, fixed